### PR TITLE
Add fancy bash prompt from orchestration workshop

### PR DESCRIPTION
**Pull request by so0k**
This bash prompt reflects `DOCKER_HOST` environment variable as added by s0ulshake to the Orchestration workshop.

It helps visualise the functionality of `docker-machine env` for students.

It was tested as follows:

```
docker build -t docker:play-with-docker-dind -f Dockerfile.dind .
docker run -it --rm -h node1 docker:play-with-docker-dind
```

result:
docker-orch-prompt

### CODE REVIEW

File: docker-prompt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+#!/bin/sh
+case "$DOCKER_HOST" in
```
Comment (marcosnils) [inline docker-prompt]: PWD doesn't use $DOCKER_HOST variable. I'm not sure if this works

Comment (marcosnils) [inline docker-prompt]: So, if I setup a swarm using docker swarm init, prompt won't change either.

Comment (so0k) [inline docker-prompt]: the requirement is for play-with-docker to run `bash` in the dind containers instead of `ash`

**Comment by so0k** [comment]
hmm, not sure if play-with-docker has `docker-machine` .. this prompt is most useful if `docker-machine` is used during the workshop

**Comment by so0k** [comment]
Thanks for feedback - while doing the orchestration workshop the prompt helped a lot in visualizing what was going on. I haven't used P-w-D enough to realise the prompt is less important here. I'm closing PR for now.

**Comment by so0k** [comment]
I noticed docker-machine was added as well as the other tools used during the workshop.

**Comment by marcosnils** [comment]
@so0k it wasn't added yet: https://github.com/franela/play-with-docker/pull/39#discussion_r88164944. 

I still don't understand what docker-machine could be used for in PWD if you can't create vm's to manage.

**Comment by so0k** [comment]
in the workshop `docker-machine` is used to manipulate the docker client target. so you can be on 1 node and talk to all the other nodes easily.

**Comment by marcosnils** [comment]
> in the workshop docker-machine is used to manipulate the docker client target. so you can be on 1 node and talk to all the other nodes easily.

Makes sense. @jpetazzo, have you used machine for this in the workshop?

**Comment by jpetazzo** [comment]
I use Machine in the workshop, but not yet with PWD. I have plans to do it in the future, though! My idea is to make sure that the containers have net-aliases `node1`, `node2`, etc., which will allow to pre-populate the Machine directory.

_/me waves at @soulshake for the awesome prompt by the way 😄  _

**Comment by marcosnils** [comment]
Let's try to merge this then. @so0k can you please rebase?

**Comment by xetorthio** [comment]
@so0k mind rebasing so we can merge this? thanks! :)

**Comment by so0k** [comment]
Done

**Comment by marcosnils** [comment]
@so0k thx for rebasing. At the end of the Dockerfile.dind we launch the default shell `sh` (https://github.com/franela/play-with-docker/blob/master/Dockerfile.dind#L35). I believe that in order for this PR to make sense we should change it to `bash` right?

**Comment by so0k** [comment]
woops, good catch, sorry - pushed that change

**Comment by marcosnils** [comment]
@so0k just tried this PR locally and it's almost there. The only annoying thing that's left is that in the current `sh` shell doing ctrl + arrow jumps to the next/prev word. After switching to bash it just prints these chars in the console:

image


@jpetazzo any ideas how to fix ths?

**Comment by so0k** [comment]
Strange, I'm on OSX using iTerm and when I do `ALT`+ arrow it jumps the words correctly.

When I use `CTRL` + arrow it doesn't print strange characters but just goes letter by letter. 

My iTerm has following key mappings though:
iterm-keymaps

**Comment by marcosnils** [comment]
@so0k I'm trying locally using PWD. I'm not using any terminal emulators so the characters are sent to `bash` directly

**Comment by so0k** [comment]
Thanks!

**Pull request merged**